GOVERNANCE.md

```diff
@@ -0,0 +1,97 @@
+# Governance
+
+`haskell-cryptography` (herein referred to as the "Organisation") is a GitHub organisation that aims to provide a support structure to open-source maintainers producing cryptography projects.
+
+The Organisation has the following goals:
+
+* Prevent situations where too few people are supporting critical projects for the ecosystem.
+* Prevent having a closed group of people having too much control over every aspect of a project.
```

Comment:
This is a bit too broad. I assume by 'a project' you meant 'a project that's ecosystemically-critical, requires consensus, etc'?